bot/processors/pennychat.py

```diff
@@ -142,6 +143,34 @@ def penny_chat_details_modal(penny_chat_invitation):
                     }
                 ]
             },
+            {
+                'block_id': 'penny_chat_visibility',
+                'type': 'input',
+                'element': {
+                    'type': 'checkboxes',
+                    'action_id': 'private_chat_select',
+                    'options': [
+                        {
+                            "text": {
+                                "type": "plain_text",
+                                "text": "private chat?",
+                            },
+                            "value": "private",
+                        },
+                        {
+                            "text": {
+                                "type": "plain_text",
+                                "text": "public chat?",
+                            },
+                            "value": "public?",
+                        },
```

Comment:
Because of the options we currently have, I don't think you should have two checkboxes here. What would happen if the user selected both? There should either be one checkbox for private (if it's checked, it is a private chat, otherwise public), or a set of radio buttons so that only one option can be selected.

Reply:
The issue is that with a checkbox field, Slack requires an input to be selected and a value returned. I haven't seen a way to get around that but I'll look a little more. All I can say right now is if I stuck with only a private checkbox, for a public chat it would never allow the user to press submit. 

Radio buttons is a good idea, looking into the documentation for that.

Reply:
Ok good to know. I did have a feeling there might be some slack weirdness behind it :P.

Reply:
Agreed w/ Nick. Somehow we've got to get away from multiple options. It really needs to be _a_ checkbox. There's got to be some way to work around this.

Reply:
https://api.slack.com/reference/block-kit/block-elements#checkboxes Here is the docs on the checkbox, I don't see any field that'll let me disable required for the section :thinking: Let me know if there's something I'm missing. I need to clean that field up and make everything capitalized anyways.

Reply:
Why not radio buttons with Public as the default? https://api.slack.com/reference/block-kit/block-elements#radio. Not as clean as a single check box, but much better!

[You can also use this to see an example](https://app.slack.com/block-kit-builder/T0JC6MASV#%7B%22blocks%22:%5B%7B%22type%22:%22section%22,%22text%22:%7B%22type%22:%22plain_text%22,%22text%22:%22Check%20out%20these%20rad%20radio%20buttons%22%7D,%22accessory%22:%7B%22type%22:%22radio_buttons%22,%22action_id%22:%22this_is_an_action_id%22,%22initial_option%22:%7B%22value%22:%22A1%22,%22text%22:%7B%22type%22:%22plain_text%22,%22text%22:%22Radio%201%22%7D%7D,%22options%22:%5B%7B%22value%22:%22A1%22,%22text%22:%7B%22type%22:%22plain_text%22,%22text%22:%22Radio%201%22%7D%7D,%7B%22value%22:%22A2%22,%22text%22:%7B%22type%22:%22plain_text%22,%22text%22:%22Radio%202%22%7D%7D%5D%7D%7D%5D,%22type%22:%22home%22%7D), and click the Actions tab to see what info you would get back. Very handy!

Reply:
@JnBrymn 

> There's got to be some way to work around this.

You forget that we are working with the Slack API ;)